spec/factories/json/json_address_factory.rb

```diff
@@ -3,6 +3,10 @@
 FactoryBot.define do
 
   factory :json_address_data, class: ::EtApi::Test::Json::Node do
+    trait :empty do
```

Comment:
We used to use an empty hash for this, but this meant that the test code was incorrectly needing to deal with hashes as well as open structs